src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java

```diff
@@ -49,17 +59,59 @@ public class MetricsBean {
 
     private final WebSocketHandler webSocketHandler;
 
+    private final ExerciseRepository exerciseRepository;
+
+    private final ExamRepository examRepository;
+
+    private final StudentExamRepository studentExamRepository;
+
+    private final CourseRepository courseRepository;
+
+    private final UserRepository userRepository;
+
+    private final StatisticsRepository statisticsRepository;
+
+    private MultiGauge activeUserMultiGauge;
+
+    private final AtomicInteger activeCoursesGauge = new AtomicInteger(0);
+
+    private final AtomicInteger coursesGauge = new AtomicInteger(0);
+
+    private MultiGauge studentsCourseGauge;
+
+    private final AtomicInteger activeExamsGauge = new AtomicInteger(0);
+
+    private final AtomicInteger examsGauge = new AtomicInteger(0);
+
+    private MultiGauge studentsExamGauge;
+
+    private MultiGauge exerciseGauge;
+
+    private MultiGauge activeExerciseGauge;
```

Comment:
Nitpick: reordering the attributes into corresponding pairs of Gauge/Integer would make it easier to read.